corehq/apps/users/models.py

```diff
@@ -3025,7 +3025,7 @@ def process_record(self, user):
                 fcm_token=self.fcm_token, fcm_token_timestamp=self.last_heartbeat, save_user=False
             )
         if save:
-            user.save(fire_signals=False)
+            user.save(fire_signals=False, update_django_user=False)
```

Comment:
The name `update_django_user` seems explanatory enough, but it isn't obvious why we would want that value set to False here and I imagine it will be less obvious a year from now. Could you add a comment explaining the thought process here?

Reply:
Yeah, sure, valid!

Reply:
How's that?

Reply:
Looks good 👍🏻